test/extended/openstack/servers.go

```diff
@@ -76,10 +79,14 @@ var _ = g.Describe("[sig-installer][Feature:openstack] The OpenStack platform",
 					instanceImage, err := images.Get(computeClient, fmt.Sprintf("%v", instance.Image["id"])).Extract()
 					o.Expect(err).NotTo(o.HaveOccurred())
 					instanceSgs := parseInstanceSgs(instance.SecurityGroups)
+					addressesMachine := getAddressesFromMachine(machine)
+					instanceAddresses, err := parseInstanceAddresses(instance.Addresses)
+					o.Expect(err).NotTo(o.HaveOccurred())
 
 					o.Expect(instanceFlavor.Name).To(o.Equal(flavorMachineSet), "Flavor not matching for instance %q", instance.Name)
 					o.Expect(instanceImage.Name).To(o.Equal(imageMachineSet), "Image not matching for instance %q", instance.Name)
 					o.Expect(instanceSgs).To(o.Equal(sgMachineSets), "SGs not matching for %q", instance.Name)
+					o.Expect(instanceAddresses).To(o.Equal(addressesMachine), "Addresses not matching for instance %q", instance.Name)
```

Comment:
Can you please mention the BZ in a comment right above this new check?

Reply:
Also, depending on your personal taste, the four lines you added can be replaced with this one-liner:

```Go
o.Expect(parseInstanceAddresses(instance.Addresses)).To(o.Equal(getAddressesFromMachine(machine))
```

the advantage would be that we have all dependent code packed together. Fewer preparatory steps above means that while reading, if I am only interested in `sgMachineSets` for example, I don't have to read about `instanceAddresses`.

but it's really up to you.